.circleci/config.base.yml

```diff
@@ -542,13 +519,40 @@ jobs:
       - restore_cache:
           keys:
             - amplify-cli-ssh-deps-{{ .Branch }}
+      - run:
+          name: Upload Pkg Binary
```

Comment:
for prod deployments, we must upload the new pkg cli _before_ we release the new version to npm. Since `lerna deploy` applies the new version number, we have to update the versions manually before generating the pkg cli.